### ISSUE TITLE
The element that opens the right side menu has the wrong role

### ISSUE THREAD

**Issue by AndyScherzinger** [nextcloud/announcementcenter#482]
...as it is a non-semantic div-element instead of a button or link.

![1](https://user-images.githubusercontent.com/1315170/177593529-12c23c1a-97b1-49ed-a309-317a4db84388.png)

The button marked in the screenshot has got the name “announcementcenter”. This is not the most user-friendly name, but there are also issues with how it communicates that it opens and closes an area. From what we can see it uses its title-attribute to communicate that it can open or close an area. This is not very robust, some assistive technologies don’t use this attribute. Use the aria-expanded attribute to communicate its status instead.

![2](https://user-images.githubusercontent.com/1315170/177593551-2209424f-773f-43a2-8b1a-189cb96aafeb.png)

We also recommend that you make this link into a button, set aria-hidden in the svg-element, and use a translatable text for the name.

**Comment by nickvergessen** [comment]
> ...as it is a non-semantic div-element instead of a button or link.

Adjusted since https://github.com/nextcloud/announcementcenter/pull/452 already

> The button marked in the screenshot has got the name “announcementcenter”.
> We also recommend that you make this link into a button, set aria-hidden in the svg-element, and use a translatable text for the name.

I don't see this happening, but then again, the button is from the Vue component and can not be controlled:
https://github.com/nextcloud/nextcloud-vue/blob/master/src/components/AppSidebar/AppSidebar.vue#L149-L155


TL;DR: Duplicate of 
https://github.com/nextcloud/announcementcenter/issues/469
